internal/service/ec2/ipam_test.go

```diff
@@ -182,6 +182,35 @@ func TestAccIPAM_cascade(t *testing.T) {
 	})
 }
 
+func TestAccIPAM_tier(t *testing.T) {
+	ctx := acctest.Context(t)
+	var ipam ec2.Ipam
+	resourceName := "aws_vpc_ipam.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(ctx, t) },
+		ErrorCheck:               acctest.ErrorCheck(t, names.EC2ServiceID),
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories,
+		CheckDestroy:             testAccCheckIPAMDestroy(ctx),
+		Steps: []resource.TestStep{
```

Comment:
Can you please add a test step with no tier defined and then perform an import to validate there is no issue

Reply:
I've added an import step to the test in 62bd959b3ecd999e9b4254bb37d90afb3f6fb457.
Regarding creating the ipam without tier config and doing an import; wouldn't the other ipam tests already cover that?